src/receipt.cpp

```diff
@@ -16,11 +16,10 @@
  */
 
 #include "receipt.hpp"
```

Comment:
There should be a newline to separate the include of the corresponding header for this unit vs other local headers vs library headers.